js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
+
+//
+// Things Warren Ellis Calls "My Little X"
+//
+var ellisMyLittleLabel = function(vocabulary) {
+    var label = 'my ';
+    switch (_.random(4)) {
+
+        case 0:
+            // eg, "my sexy mammals of the internet"
+            label += _.sample(vocabulary.sexy) + ' ';
+            label += _.sample(vocabulary.mammals) + ' of ';
+            if (Math.random() < 0.5) {
```

Comment:
No good reason, really, other than `Math.random()` was what I found when I looked up how to do a 50/50 chance.

I suppose I could change it to:

```javascript
switch (_.random(1)) {
    case 0:
    // option A
    break;
    default:
    // option B
}
```

Is there any reason to prefer one over the other? `switch` statements seem like overkill for two options, but maybe just for consistancy?

Oh wait, I see, you're not objecting to the `if` statement, you're objecting to using `Math.random()` instead of `_.random()`.

That's a good point, I'll update to standardize on underscore's random function.